### DISCUSSION TITLE
why is the molecule object distorted after being loaded with SDMolSupplier?

### DISCUSSION THREAD

**Discussion by LiuCMU** [rdkit/rdkit#6125] [Q&A] [answered]
I tried to visualize a molecu in Jupyter Notebook after loading it with `Chem.SDMolSupplier`. But the visualization does not make sense to me. Here is the code to reproduce:

```
from rdkit import Chem


source = 'PAYRUJLWNCNPSJ-UHFFFAOYNA-N.sdf'
mol = Chem.SDMolSupplier(source)[0]
mol
```
The molecule looks like this:
![Screen Shot 2023-02-23 at 12 00 42 PM](https://user-images.githubusercontent.com/60156077/220977387-a9ebbd8f-a2b2-45e6-9405-1ad5825b62b6.png)

When I visualize the SDF file, it's simply an aniline. Why would it look so distorted with RDKit? I have been sticking here for a morning. Please let me know if you know how to tackle it. Thank you very much!
![Screen Shot 2023-02-23 at 12 06 46 PM](https://user-images.githubusercontent.com/60156077/220978942-b5864c9f-556f-499f-9444-91e8f92c0c7a.png)

I attached the source SDF file. Please note that the file extension was changed to txt for GitHub. 
[PAYRUJLWNCNPSJ-UHFFFAOYNA-N.txt](https://github.com/rdkit/rdkit/files/10816213/PAYRUJLWNCNPSJ-UHFFFAOYNA-N.txt)

My RDKit version: 2022.03.2 py39h89e00b9_0    conda-forge
My system information: Ubuntu 18.04.5 LTS

**Comment by greglandrum** [comment, marked as answer]
@LiuCMU your molecule has 3D coordinates. What you are seeing is a direct 2D rendering of those 3D coordinates.
If you want to see 3D conformers in the jupyter notebook, you can set `IPythonConsole.ipython_3d = True` (make sure that you have the `py3dmol` package installed)
or, if you want to generate 2D coordinates for the molecule you can do:
```
from rdkit.Chem import rdDepictor
rdDepictor.Compute2DCoords(m)
```
